quickstart/javascript/browser/intent-recognition/README.md

```diff
@@ -7,7 +7,7 @@ This sample shows how to recognize intent with the Speech Service using the Spee
 
 ## Prerequisites
 
-* A LUIS account. You can get one for free through the [LUIS portal](https://www.luis.ai/home).
+* A LUIS account. [Create a new Azure account, and try Cognitive Services for free.](https://azure.microsoft.com/free/cognitive-services/)
```

Comment:
this change should probably be reverted. the original is about a LUIS account not an azure subscription. thanks!